### PR TITLE
Code not required

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: core/exception/args_spec.rb [hunk 1]
```diff
@@ -1,5 +0,0 @@
-require_relative '../../spec_helper'
-
-describe "NoMethodError#args" do
-  it "needs to be reviewed for spec completeness"
-end
```
Comment (BurdetteLamar) [inline core/exception/args_spec.rb]: Covered at no_method_error_spec.rb.

File: core/exception/arguments_spec.rb [hunk 2]
```diff
@@ -1,11 +0,0 @@
-require_relative '../../spec_helper'
-
-describe "ArgumentError" do
-  it "is a subclass of StandardError" do
-    StandardError.should be_ancestor_of(ArgumentError)
-  end
```
Comment (BurdetteLamar) [inline core/exception/arguments_spec.rb]: Covered by hierarchy_spec.rb.

File: core/exception/arguments_spec.rb [hunk 3]
```diff
@@ -1,11 +0,0 @@
...
-
-  it "gives its own class name as message if it has no message" do
-    ArgumentError.new.message.should == "ArgumentError"
-  end
```
Comment (BurdetteLamar) [inline core/exception/arguments_spec.rb]: ArgumentError is the only Exception for which this is tested.  All, or none?

Comment (eregon) [reply core/exception/arguments_spec.rb]: Yeah, this is likely already tested in Exception#message specs.

**Pull request merged**